ui/articles-aem-renderer/src/main/AndroidManifest.xml

```diff
@@ -1,2 +1,11 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="org.cru.godtools.article.aem" />
+    xmlns:tools="http://schemas.android.com/tools"
+    package="org.cru.godtools.article.aem" >
+    <application>
+        <activity
+            android:name="org.cru.godtools.articles.aem.activity.AemArticleItemActivity"
+            android:theme="@style/Theme.GodTools.Tool.AppBar"
+            android:parentActivityName="org.cru.godtools.article.activity.ArticlesActivity"
+            tools:targetApi="jelly_bean" />
```

Comment:
you also need to include the parent activity meta-data xml node so that it works on ICS